content/ja/handbook/marketing/brand-and-product-marketing/product-and-solution-marketing/demo/gke-setup.md

```
@@ -0,0 +1,238 @@
+ </iframe>
+</figure>
+
+## 準備
+
+- Google Cloud Platform アカウントが必要です。GitLab 従業員はこれを保有しています。GitLab アカウントでログインしていることを確認してください。
+- [Google Kubernetes Engine](https://console.cloud.google.com/kubernetes) にログインします。
+- GitLab 従業員は `gitlab-demos` プロジェクトを使用してください。それ以外の方は、作業するプロジェクトを選択または作成してください。
+
+  - URL: [https://console.cloud.google.com/kubernetes/list?project=gitlab-demos](https://console.cloud.google.com/kubernetes/list?project=gitlab-demos)
+
+- 以前にデモを実行したものの[デモクラスタ](https://console.cloud.google.com/kubernetes/list)をクリーンアップしていない場合は、[今すぐクリーンアップしてください](#cleanup)。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)